### DISCUSSION TITLE
Reg: python code

### DISCUSSION THREAD

**Discussion by Pilli68** [jsvine/pdfplumber#670] [Q&A]
could you please tell me your entire project python code file name please.

**Comment by jsvine** [comment]
Unfortunately, I don't understand this question. Can you clarify?

**Reply by Pilli68** [reply]
@ jsvine ,my requirement meet 100% to your project. So i need your full  python script/code.

**Reply by jsvine** [reply]
This is an open-source project, so you can find all the code here: https://github.com/jsvine/pdfplumber

<img width="419" alt="Screen Shot 2022-06-15 at 9 31 18 AM" src="https://user-images.githubusercontent.com/534702/173839611-00454e08-0389-4d7b-9bac-0b29cbe2c642.png">
